wallets-testing/services/service.widget.ts

```diff
@@ -0,0 +1,102 @@
+import { ETHEREUM_WIDGET_CONFIG } from '../config';
+import { WalletTypes } from '@lidofinance/wallets-testing-wallets';
+import { Page, test, expect } from '@playwright/test';
+import { BrowserService } from '@lidofinance/browser-service';
+import { WidgetPage } from '../pages/widget.page';
+import { waitForTextContent } from '../utils/helpers';
+
+export interface TxConfig {
+  txAmount: string;
+  tokenAddress?: string;
+  mappingSlot?: number;
+}
+
+export class WidgetService {
+  page: Page;
+  widgetPage: WidgetPage;
+
+  constructor(private browserService: BrowserService) {
```

Comment:
I suppose you can pass `browserService.getBrowserContextPage():Page` since BrowserService is too much